.coderabbit.yaml

```
@@ -0,0 +1,104 @@
+  - "**/ApiClient.java"
+  - "**/ApiException.java"
+  - "**/Configuration.java"
+  - "**/JSON.java"
+  - "**/ApiResponse.java"
+  - "**/Pair.java"
+  - "**/StringUtil.java"
+  - "**/TypeRef.java"
+  - "**/org/openapitools/**"
+  - "**/build/generated-sources/openapi/**"
+  - "**/src/gen/java/main/**"
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)